### PR TITLE
Feat(TaxCenter): add report modal

### PR DESCRIPTION
**Pull request by hcondori-bc**
## Description (optional)
Add generate report modal

## Testing Steps (optional)

tax-center-modal

### CODE REVIEW

**Review by cmonti-bc — APPROVED** [review]

**Review by milan-bc — APPROVED** [review]
lgtm

File: packages/blockchain-wallet-v4-frontend/src/modals/GenerateReport/template.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  StyledModalFooter,
+  StyledModalHeader,
+  Title
+} from './model'
+
+const GeneratReport = ({ onClose }) => (
+  <Modal size='medium'>
+    <StyledModalHeader onClose={onClose} />
+    <StyledModalBody>
+      <Image name='subtract-check-circle' width='88px' height='88px' />
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/modals/GenerateReport/template.tsx:19]: Should the loader on the image be animated? I think it's just a static image right now?

Comment (hcondori-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/GenerateReport/template.tsx:19]: Yes, is a static image. Should I change anything? Feel free to suggest

Comment (hcondori-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/GenerateReport/template.tsx:19]: I will talk with UX about the icon. I think look great with some animation, but can be part of another pr

**Pull request merged**